src/Skeletor/Packages/Package.php

```diff
@@ -135,9 +136,46 @@ public function setFacade(string $facade)
         $this->facade = $facade;
     }
 
+    /**
+     * @return array
+     */
+    public function getEnvironmentVariables()
+    {
+        return $this->envVariables;
+    }
+
+    /**
+     * @param array $vars
+     */
+    public function setEnvironmentVariables(array $envVariables)
+    {
+        $this->envVariables = $envVariables;
+    }
+
+    /**
+     * @return boolean
+     */
+    public function hasEnvironmentVariables()
+    {
+        return count($this->envVariables) > 0;
+    }
+
+    /**
+     * Install the composer package
+     */
     public function install()
     {
         $command = $this->composerManager->preparePackageCommand($this->getInstallSlug(), $this->getVersion(), $this->getPackageOptions());
         $this->composerManager->runCommand($command);
     }
+
+    /**
+     * Publish the configuration of the package
+     *
+     * @return string
+     */
+    public function publishConfig()
+    {
+        return shell_exec(sprintf('php artisan vendor:publish --provider="%s"', $this->getProvider()));
+    }
```

Comment:
@kielabokkie you could use the `runCommand()` method in the `ComposerManager`, this will give you a nice API the execute shell commands.  I only think we need to move the `runCommand()` method to a different class, because it shouldn't rely on the composer class.